abb_irb4400_support/CMakeLists.txt

```diff
@@ -6,7 +6,12 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
+if (CATKIN_ENABLE_TESTING)
+  find_package(roslaunch REQUIRED)
+  roslaunch_add_file_check(tests/roslaunch_test.xml)
```

Comment:
Please add a `DEPENDENCIES` argument to the `roslaunch_add_file_check(..)` invocation. See [here](https://github.com/ros-industrial/fanuc/blob/5e2cf28d4b65af0724566a132c0f56aff6f737d2/fanuc_m10ia_support/CMakeLists.txt#L11-L18) for an example.